portlets/src/main/webapp/vue-app/commons/components/Widget.vue

```diff
@@ -52,7 +55,7 @@ export default {
     },
     height: {
       type: String,
-      default: () => '338px',
+      default: () => '358px',
```

Comment:
Should we change the default `height` ? If yes, why ?

Reply:
because We have this UI glitch
![Screenshot from 2023-10-19 14-15-28](https://github.com/Meeds-io/gamification/assets/40363501/430168b3-1924-4721-8d82-bece71c56756)